### ISSUE TITLE
[device-port] [bardock pro] [Halium-9.0] BQ Aquaris X Pro

### ISSUE THREAD

**Issue by stefanomelchior** [Halium/projectmanagement#125]
Tree: **halium-9** 

- [X] Create manifest 
- [X] Boot image and system image build successfully
- [X] Device boots into rootfs, `usb: Manufacturer: GNU/Linux Device` appears in `dmesg` on host.
- [X] LXC container starts and does not crash
- [ ] libhybris tests
  * [ ] test_gps
  * [ ] test_hwcomposer
  * [ ] test_lights
  * [ ] test_vibrator
  * [ ] test_wifi
  * [ ] test_sensors
  * [ ] test_audio
  * [ ] test_camera
  * [ ] test_input
  * [ ] test_nfc
  * [ ] test_recorder

**Currently**
Able to connect via ssh to UT, and lxc container starts when deleting /etc/init/lxc-android-config.override (needed for peeble devices).
It boots from `systempart` (`/dev/mmcblk0p24`), so it is OTA compliant.

Files:
[bardock_defconfig](https://github.com/Halium/projectmanagement/files/5155713/bardock_defconfig.txt)
[bq_bardockpro.xml vers.20200901](https://github.com/Halium/projectmanagement/files/5155716/bq_bardockpro.xml_20200901.txt)
[fixup-mountpoints-bardockpro vers. 20200901](https://github.com/Halium/projectmanagement/files/5155718/fixup-mountpoints-bardockpro.txt)

Repos where I'm working on:
Kernel (branch: guf-bardock): [kernel](https://github.com/stefanomelchior/android_kernel_bq_msm8953)
Device setup (branch: guf-bardock): [device](https://github.com/stefanomelchior/android_device_bq_msm8953-common/)

**Comment by Flohack74** [comment]
@stefanomelchior can you tell us if you have abandoned this port, and what has happened so far?

**Comment by stefanomelchior** [comment]
No, at all. I only trying to end up tenshi porting for UT, then get back to bardock.
Noteworthy bardock is Android-8 based, so it needs halium-8.1 or later support

**Comment by luksus42** [comment]
@stefanomelchior I just watched the device sources of this port, to learn from it... Can you please tell me, where the system/halium sources come from? Is that something special for this device or something generic?

**Comment by stefanomelchior** [comment]
@luksus42 you need to build from sources (starting from lineage-16.0 kernel and device) and use ubuntu-touch-android9.tar.gz rootfs image to install OS. At this extent, I am struggling on GUI to start [logcat output when running ```EGL_PLATFORM=hwcomposer test_hwcomposer```](https://paste.ubuntu.com/p/MwXCShdt7G/) and dmesg is flooded by android.services starting and kill:
```[ 6701.098341] init: Sending signal 9 to service 'vendor.camera-provider-2-4' (pid 6790) process group... HYBRIS: killing PID instead of process group.
[ 6706.053602] init: Sending signal 9 to service 'vendor.bluetooth-1-0-qti' (pid 6798) process group... HYBRIS: killing PID instead of process group.
[ 6706.068244] init: Sending signal 9 to service 'vendor.vibrator-1-0' (pid 6800) process group... HYBRIS: killing PID instead of process group.
[ 6706.100125] init: Sending signal 9 to service 'vendor.camera-provider-2-4' (pid 6799) process group... HYBRIS: killing PID instead of process group.```
